src/style.css

```diff
@@ -1 +1,67 @@
-/* Styles go here */
+html {
+  font-family: "Roboto", sans-serif;
+  font-style: medium;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+body {
+  margin: 0;
+}
+
+ul {
+  padding: 0;
+}
+
+.home {
+  display: flex;
+  justify-content: space-between;
+}
+
+.logo {
+  margin: 10px;
+  margin-left: 50px;
```

Comment:
Looks strange, better use just one rule to set margins

Reply:
Also if you will use `align-items` to header, there will be no need in vertical margins